pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -218,15 +223,26 @@ func dumpInstallStrategyToBytes(strategy *InstallStrategy) []byte {
 	return b.Bytes()
 }
 
+func patchValidation(crd *extv1beta1.CustomResourceDefinition, name string) *extv1beta1.CustomResourceDefinition {
+	if name != "vmi" && name != "vm" {
```

Comment:
why filter out non vmi/vm crds here? We want all our objects to have a schema.

Reply:
Had some problem with validation so It would introduce some risk. This seems to be gone with the newer version controller-gen.